include/javascript/mozaik/jquery.mozaik.css

```diff
@@ -230,4 +230,22 @@ div.mozaik-preloader {
 .mozaik-thumbs {
     background: #fff;
     height: 400px;
+}
+
+/* margin set */
+.mozaik-width-set {
+    width: 100%;
+    float: none;
+    display: block;
+    height: 0px;
```

Comment:
This should be set to 0 not 0px. The px is not needed.

Should this be display: none?